resources/user-guides/mine-to-pool.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: static_page
+title: How to mine on a pool
+---
+# Selecting a pool
+
+There are many pools to choose from, a list is available at
+[moneropools.com](https://moneropools.com). Mining on a larger pool could mean
+more frequent payouts, but mining on a smaller pool helps to keep the network
+decentralised.
+
+# Selecting a CPU miner
+
+Just like pools, there are a lot of miners to choose from. The one that you
+should pick depends on the hardware you want to mine on. This guide will only
+use a CPU miner, and will be using
+[xmr-stak-cpu](https://github.com/fireice-uk/xmr-stak-cpu).
```

Comment:
It shouldn't be a problem, but might be a good idea to nominate a couple of alternatives with links, like Cpuminer [tpruvot](https://github.com/tpruvot/cpuminer-multi/tree/master) and [Wolf](https://github.com/OhGodAPet/cpuminer-multi)